### PR TITLE
Add method for translating FHIR codeSystems to PHDC-compliant codeSystems

### LINKED ISSUE #1329: Correct the value of `codeSystem` inside of the `<code>` tag within Clinical Information
## Action Requested
The `codeSystem` value needs to be changed from:
- `http://loinc.org` -> `LOINC`

Here is the current FHIR to PHDC conversion based on recently built image of our message parser:
```xml
<title>Clinical Information</title>
          <entry typeCode="COMP">
            <observation classCode="OBS" moodCode="EVN">
              <code code="22322-2" codeSystem="http://loinc.org"
                displayName="Hepatitis A virus IgM antibody [Presence] in Serum by Immunoassay" />
              <value />
            </observation>
          </entry>
```

And this is what it should look like:
```xml
<title>Clinical Information</title>
          <entry typeCode="COMP">
            <observation classCode="OBS" moodCode="EVN">
              <code code="22322-2" codeSystem="LOINC"
                displayName="Hepatitis A virus IgM antibody [Presence] in Serum by Immunoassay" />
              <value />
            </observation>
          </entry>
```

## Acceptance Criteria
When,
- the `codeSystem` value is correctly displaying `LOINC` as its value in a response to our Sample FHIR bundle request when sent to the `fhir_to_phdc` endpoint; and,
- the tests are passing locally and via GitHub actions

## Additional Information
Here is the validation error:
```
Element '{urn:hl7-org:v3}code', attribute 'codeSystem': 'http://loinc.org' is not a valid value of the
union type '{urn:hl7-org:v3}uid' .
```

### PR DESCRIPTION
**Pull request by m-goggins**
# PULL REQUEST

## Summary
This PR adds a new helper method to ensure that the codeSystems coming out of a FHIR bundle, e.g., `http://loinc.org`, are translated to the appropriate PHDC-compliant code system data, e.g., `LOINC`. 

## Related Issue
Fixes #1329 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Review by DanPaseltiner — COMMENTED** [review]
Thanks for putting this together! Curious to hear your thoughts on switching to `getattr` (comment below).

File: containers/message-parser/app/phdc/builder.py [hunk 1]
```diff
@@ -504,27 +506,107 @@ def _sort_observation(self, observation: Observation) -> Observation:
...
+        """
+        Translates FHIR-specific codeSystem(s) within an Observation to PHDC-compliant
+        codeSystem(s) and codeSystemName(s).
+
+        :param observation: The data for building the observation element as an
+            Entry object.
+        :return: The data for building the observation element as an
+            Entry object with update codeSystemName(s).
+        """
+        translated_observation = copy.deepcopy(observation)
```
[the remark this answers is not in the document]
Comment (m-goggins) [reply containers/message-parser/app/phdc/builder.py:543]: Oops, that was leftover from a previous iteration of work

File: containers/message-parser/app/phdc/builder.py [hunk 2]
```diff
@@ -504,27 +506,107 @@ def _sort_observation(self, observation: Observation) -> Observation:
...
+                "codeSystem": "2.16.840.1.114222.4.5.1",
+            },
+        }
+
+        cs_values = {
+            "code_system": observation.code_system,
+            "code_code_system": observation.code_code_system,
+            "value_qualitative_code_system": observation.value_qualitative_code_system,
+            "value_quant_code_system": observation.value_quant_code_system,
+        }
```
[the remark this answers is not in the document]
Comment (m-goggins) [reply containers/message-parser/app/phdc/builder.py]: Nice 😎 much more succinct

**Review by DanPaseltiner — APPROVED** [review]
🚀

**Pull request merged**